Formula/makensis.rb

```diff
@@ -32,8 +39,13 @@ def install
     # https://sourceforge.net/p/nsis/bugs/1085/
     ENV.libstdcxx if ENV.compiler == :clang
 
+    # requires zlib (win32) to build utils
+    resource("zlib-win32").stage do
```

Comment:
Well I tried that and for some reason I couldn't comprehend, the configure script failed to recognize zlib.  So unless you can get it to work, maybe we'll just leave it as-is.